src/metrics.rs

```diff
@@ -1,30 +1,137 @@
-use metrics::{Counter, Histogram};
+use std::time::Duration;
+
+use metrics::{counter, histogram, Counter, Histogram};
 use metrics_derive::Metrics;
 
+use crate::server::PayloadSource;
+
 #[derive(Metrics)]
 #[metrics(scope = "rpc")]
 pub struct ServerMetrics {
-    #[metric(describe = "Count of forkchoice_updated_v3 calls proxied to the builder")]
-    pub fcu_count: Counter,
+    #[metric(describe = "Total latency for server `engine_newPayloadV3` call")]
+    pub new_payload_v3_total: Histogram,
```

Comment:
```suggestion
    pub new_payload_v3: Histogram,
```
I prefer the name without total prefix

Reply:
Ideally there should be some indication of the metric's use in the name to help distinguish

Reply:
I think we can have a separate namespace between the client and total latencies, but its not as obvious as adding a total latency name to help distinguish the different metric